aspnetcore/blazor/components.md

```diff
@@ -152,19 +152,19 @@ Unlike `onchange`, `oninput` fires for every character that is input into the te
 Data binding works with <xref:System.DateTime> format strings. Other format expressions, such as currency or number formats, aren't available at this time.
 
 ```cshtml
-<input bind="@StartDate" format-value="yyyy-MM-dd" />
+<input @bind="@StartDate" @bind:format="yyyy-MM-dd" />
 
 @code {
     [Parameter]
     private DateTime StartDate { get; set; } = new DateTime(2020, 1, 1);
 }
 ```
 
-The `format-value` attribute specifies the date format to apply to the `value` of the `input` element. The format is also used to parse the value when an `onchange` event occurs.
+The `@bind-value:format` attribute specifies the date format to apply to the `value` of the `input` element. The format is also used to parse the value when an `onchange` event occurs.
```

Comment:
Is this right? this has `@bind-value:format` and the code sample has `@bind:format`